docs/src/main/java/io/helidon/docs/se/openapi/OpenApiGeneratorSnippets.java

```diff
@@ -0,0 +1,386 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.docs.se.openapi;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.helidon.http.Status;
+import io.helidon.webclient.api.HttpClientResponse;
+import io.helidon.webclient.api.WebClient;
+import io.helidon.webclient.api.WebClientConfig;
+import io.helidon.webserver.http.ServerRequest;
+import io.helidon.webserver.http.ServerResponse;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class OpenApiGeneratorSnippets {
+
+    /*
+    To generate a project containing the types which need to be mocked--in case of changes in the upstream generator:
+
+    1. Download the generator: see [these instructions](https://github.com/OpenAPITools/openapi-generator?tab=readme-ov-file#13---download-jar).
+    2. `mkdir seServer`
+    3. `cd seServer`
+    4. ```java
+       java -jar {downloadLocation}/openapi-generator-cli.jar generate \
+       -g java-helidon-server \
+       --library se \
+       -i ~/mic/j4c/helidon/docs/src/main/resources/petstorex.yaml \
+       -p useAbstractClass=true \
+       --helidonVersion x.y.z
+       ```
+       where `x.y.z` is the version of Helidon you are working with.
+    5. `mkdir ../seClient`
+    6. `cd ../seClient`
+    7. ```java
+       java -jar {downloadLocation}/openapi-generator-cli.jar generate \
+       -g java-helidon-client \
+       --library se \
+       -i ~/mic/j4c/helidon/docs/src/main/resources/petstorex.yaml \
```

Comment:
`~/mic/j4c/helidon` is specific to your system.

Reply:
Fixed.